packages/core/src/types/Uint8ArrayType.ts

```diff
@@ -1,8 +1,6 @@
-import { Type } from './Type';
-import type { Platform } from '../platforms';
-import type { EntityProperty } from '../typings';
+import { BlobType } from '.';
```

Comment:
this could cause unnecessary import cycles
```suggestion
import { BlobType } from './BlobType';
```